documentation/modules/exploit/linux/local/ueb_priv_esc.md

```diff
@@ -0,0 +1,38 @@
+## Vulnerable Application
+
+  Unitrends UEB 9/10 local privesc
+
+  This exploit leverages bpserverd proprietary protocol to issue commands
+  as root.
+
+## Verification Steps
+
+  1. Get a shell with exploit/linux/http/ueb10_api_systems
+  2. ```use exploit/linux/local/ueb_priv_esc ```
+  3. ```set session [SESSION]```
+  4. ```exploit```
+  5. A highpriv meterpreter session should have been opened successfully
+
+## Scenarios
+
+### UEB 10.0 on CentOS 6.5
```

Comment:
You could add an additional scenario with https://github.com/rapid7/metasploit-framework/pull/9946#issuecomment-429564158 showing 9.2